src/HttpHdrScTarget.h

```diff
@@ -30,13 +32,8 @@ class HttpHdrScTarget
     static const int MAX_AGE_UNSET=-1; //max-age is unset
     static const int MAX_STALE_UNSET=0; //max-stale is unset
 
-    HttpHdrScTarget(const char *target_):
-        mask(0), max_age(MAX_AGE_UNSET), max_stale(MAX_STALE_UNSET),target(target_) {}
-    HttpHdrScTarget(const String &target_):
-        mask(0), max_age(MAX_AGE_UNSET), max_stale(MAX_STALE_UNSET),target(target_) {}
-    HttpHdrScTarget(const HttpHdrScTarget &t):
-        mask(t.mask), max_age(t.max_age), max_stale(t.max_stale),
-        content_(t.content_), target(t.target) {}
+    explicit HttpHdrScTarget(const char *target_) : target(target_) {}
+    explicit HttpHdrScTarget(const String &target_) : target(target_) {}
 
```

Comment:
If the existing code copies HttpHdrScTargets, then please explicitly default the copy constructor (if possible):

```suggestion

    explicit HttpHdrScTarget(const HttpHdrScTarget &) = default;

```

If the existing code does not copy HttpHdrScTargets, then please explicitly delete this copy constructor:

```suggestion

    explicit HttpHdrScTarget(const HttpHdrScTarget &) = delete; // avoid accidental string copies

```

The latter is the preferred solution if it is viable, of course.

Reply:
Done in e8df08